src/scss/components/radio/_index.scss

```diff
@@ -0,0 +1,265 @@
+@import  "vars";
+@import  "components/form/index";
```

Comment:
Please remove import from line 2.

We're already importing the needed mixins on lines 3-4.

Reply:
Changes have been done, please review.